message_definitions/v1.0/uAvionix.xml

```diff
@@ -118,5 +118,13 @@
       <description>Transceiver heartbeat with health report (updated every 10s)</description>
       <field type="uint8_t" name="rfHealth" enum="UAVIONIX_ADSB_RF_HEALTH" display="bitmask">ADS-B transponder messages</field>
     </message>
+    <message id="10004" name="UAVIONIX_ADSB_OUT_CFG_REGISTRATION">
+      <description>Aircraft Registration. This is often referred to as the callsign for ADSB-Out vehicles.</description>
+      <field type="char[9]" name="registration">Aircraft Registration (ASCII string A-Z, 0-9 only), e.g. “N8644B “. Trailing spaces (0x20) only. This is null-terminated.</field>
```

Comment:
can i request the removal of utf8 characters?

Reply:
yup, those snuck in from a pdf copy/paste. Just removed them. thanks!